### PR TITLE
Add details for 4/22 ingress-nginx CVEs.

### PR DESCRIPTION
**Pull request by cjcullen**
(also fix up the time of the previous ingress-nginx DATE_PUBLIC)

### CODE REVIEW

**Comment by cjcullen** [comment]
Test failure is the (expected) naming warning:
```
 WARNING: expected .affects.vendor.vendor_data[0].product.product_data[0].product_name to be 'Kubernetes'; found 'Kubernetes ingress-nginx'
 ```

**Comment by enj** [comment]
> Test failure is the (expected) naming warning:
> 
> ```
>  WARNING: expected .affects.vendor.vendor_data[0].product.product_data[0].product_name to be 'Kubernetes'; found 'Kubernetes ingress-nginx'
> ```

@cjcullen is this something we need to address before merging?  Otherwise seems fine?

/lgtm

File: 2021/25xxx/CVE-2021-25745.json [hunk 1]
```diff
@@ -1,18 +1,101 @@
...
+            {
+                "refsource": "MISC",
+                "url": "https://groups.google.com/g/kubernetes-security-announce/c/7vQrpDZeBlc",
+                "name": "https://groups.google.com/g/kubernetes-security-announce/c/7vQrpDZeBlc"
+            },
+            {
+                "refsource": "MISC",
+                "url": "https://github.com/kubernetes/ingress-nginx/issues/8502",
+                "name": "https://github.com/kubernetes/ingress-nginx/issues/8502"
+            },
```
Comment (tallclair) [inline 2021/25xxx/CVE-2021-25745.json:92]: trailing comment. json :roll_eyes:

**Review by tallclair — COMMENTED** [review]
We don't need to address the warning, but the parse error is a problem

```
parse error: Expected another array element at line 93, column 9
```

File: 2021/25xxx/CVE-2021-25745.json (hunk 1 above)
Comment (cjcullen) [reply 2021/25xxx/CVE-2021-25745.json]: Ugggh, vscode even caught it for me. Fixed.

**Pull request merged**